editor/plugins/script_text_editor.cpp

```diff
@@ -910,7 +910,7 @@ void ScriptTextEditor::_edit_option(int p_op) {
 				tx->set_line_as_breakpoint(line, dobreak);
 				ScriptEditor::get_singleton()->get_debugger()->set_breakpoint(script->get_path(), line + 1, dobreak);
 			}
-		}
+		} break;
```

Comment:
Note: I'm changing the logic here, but the previous code did not seem like this fallthrough was intentional. I haven't tested (yet) to confirm.